Makefile

```diff
@@ -52,6 +55,11 @@ test:
 	export PYTHONPATH=./backend:/./tests && \
 	python -m pytest -v -ra tests/
 
+integration-tests: upgrade-pip install-backend install-cdkproxy install-integration-tests
```

Comment:
why do we need install-backend and cdkproxy?

Reply:
It is related to one of your comments below. When we use the `ignore_module_tests_if_not_active` method to get what is. loaded and what's not we initialize the dataall package and we import all subpackages some of which depend on the packages installed. We would not have this issue if we check if modules are enabled/disabled in a different way.

What I can do is to remove `ignore_module_tests_if_not_active` because at the moment it is not needed (there are tests only for organization) and think of a different way of implementing module test switch. wdyt?